### DISCUSSION TITLE
[react-markdown] Using react-markdown with AWS amplify

### DISCUSSION THREAD

**Discussion by JoseMLopezSanz** [remarkjs/remark#773] [Q&A] [answered]
I've been using react-markdown and it has been working perfectly on my local set up. But once my app is deployed to AWS amplify, the component just renders plain HTML as a string instead. Has anyone successfully used react-markdown with a AWS amplify deployment?

**Comment by ChristianMurphy** [comment, marked as answer]
I have used create-react-app + aws amplify console + react-markdown before and it's worked fine.

**Comment by lopezdp** [comment]
having same problem - you ever find a solution? I have another app deployed on amplify using `/with-html` on the v4 of react-markdown but v9 is different and no longer supports that so am figuring out how to deploy as it works great locally.

ive tried configuring redirects with no luck

**Reply by ChristianMurphy** [reply]
Welcome @lopezdp! 👋 
This issue is 3 years old, and several major versions of Amplify and `react-markdown` behind.
A new discussion with full context would likely better serve you.
If you (@lopezdp) or the original poster (@JoseMLopezSanz) would like more advice.
Take some time to frame your problem https://github.com/remarkjs/.github/blob/main/support.md

Include:
* what your goal is in using `react-markdown` is in your app (the remark ecosystem offers several options to render to react, another could be better suited to your use-cases)
* Share what versions of libraries you are using
* Share code samples of what you have tried

**Reply by ChristianMurphy** [reply]
It's also worth reminding folks, that Amplify has it's own support community https://discord.com/invite/amplify which can help with platform specific issues.
